config/providers/openstack/common-config.yaml

```diff
@@ -0,0 +1,46 @@
+openstackMonitoring:
+  enabled: true
+storageClasses:
+  default: cinder-csi
+networkPolicies:
+  kubeSystem:
+    openstack:
+      enabled: true
+      ips:
+        - set-me
+      ports:
+        - 5000
+        - 8774
+        - 8776
+  global:
+    externalLoadBalancer: false
+    ingressUsingHostNetwork: false
+  ingressNginx:
+    enabled: true
+    ingressOverride:
+      enabled: false
+objectStorage:
+  type: s3
+  s3:
+    region: set-me
+    regionEndpoint: set-me
+    forcePathStyle: true
+ingressNginx:
+  controller:
+    config:
+      useProxyProtocol: false
+    useHostPort: false
+    service:
+      enabled: true
+      type: LoadBalancer
+      annotations: {}
+      allocateLoadBalancerNodePorts: true
+externalTrafficPolicy:
+  local: false
+opa:
+  rejectLoadBalancerService:
+    enabled: false
+calicoAccountant:
+  backend: iptables
```

Comment:
Currently this is set by the `k8s-installer` config as well, maybe we should make a decision on where this should be set, since the `backend` is dependent on the OS version used :thinking:

Reply:
And the OS version is decided by the installer? Makes sense to me!

Reply:
3d folder, with base image flavor? So one for ubuntu-2004 and one for 2204?

Reply:
I think we can keep it in the `installer` layer for now and make the assumption that for `capi` we use nftables (it will be added as a platform administrator notice in my PR), as we only have one variable dependent on the OS version. If there will be more in the future I think it could make sense to add an additional flavor.